### PR TITLE
#42 generate volunteer scaffold

### LINKED ISSUE #14: Volunteer CRUD
Volunteers are who we will send solicit-help emails to, and assign Attendances to
Follow data model diagram for fields and relationships

### PR DESCRIPTION
**Pull request by gabezurita**
Resolves #14 

### Type of change
* New feature (non-breaking change which adds functionality)

### How Has This Been Tested?
No tests needed yet, just a scaffold. As we develop further, we can write tests.

### CODE REVIEW

File: app/models/volunteer.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Volunteer < ApplicationRecord
+end
```
Comment (rjisoo) [inline app/models/volunteer.rb:15]: @gabezurita don't we have to create Volunteer model as well? I don't see separate ticket for creating table

Comment (jgaskins) [reply app/models/volunteer.rb:15]: @rjisoo Nice catch!

@gabezurita Can you run `rake db:migrate` and push the change to `db/schema.rb`?

Comment (gabezurita) [reply app/models/volunteer.rb:15]: Done!

Comment (gabezurita) [reply app/models/volunteer.rb:15]: There were also changes to the fields we wanted for the volunteer model, so redid the scaffold to include those changes.

**Review by mirandawang — APPROVED** [review]

**Review by abachman — APPROVED** [review]
👍

**Pull request merged**